plugin/web/extensions/sidebar/modals/welcomescreen/index.tsx

```diff
@@ -0,0 +1,208 @@
+import { styled } from "@web/theme";
+
+const WelcomeScreen: React.FC = () => {
+  const {
+    ShowVideo,
+    handleDontShowAgain,
+    dontShowAgain,
+    handleShowVideo,
+    handleClose,
+    handleCloseVideo,
+  } = useHooks();
+
+  return (
+    <Wrapper>
+      {!ShowVideo ? (
+        <>
+          <CloseButton>
+            <Icon size={32} icon="close" onClick={handleClose} />
+          </CloseButton>
+          <InnerWrapper>
+            <TryMapWrapper>
+              <TextWrapper width={192} height={46}>
+                <Text weight={700} size={48}>
+                  ようこそ
+                </Text>
+              </TextWrapper>
+              <ImgSection>
+                <TextWrapper width={215} height={32}>
+                  <Text weight={500} size={20}>
+                    マップを使ってみる
+                  </Text>
+                </TextWrapper>
+                <ImgWrapper>
+                  <img src={welcomeScreenVideo} onClick={handleShowVideo} />
+                </ImgWrapper>
+              </ImgSection>
+            </TryMapWrapper>
+            <BtnsWrapper>
+              <ButtonWrapper>
+                <TextWrapper width={84} height={21}>
+                  <Text weight={500} size={14}>
+                    ヘルプをみる
+                  </Text>
+                </TextWrapper>
+              </ButtonWrapper>
+              <ButtonWrapper>
+                <Icon size={20} icon="plusCircle" color="#fafafa" />
+                <TextWrapper width={84} height={21}>
+                  <Text weight={500} size={14}>
+                    ヘルプをみる
+                  </Text>
+                </TextWrapper>
+              </ButtonWrapper>
+            </BtnsWrapper>
+          </InnerWrapper>
+          <CheckWrapper>
+            <Checkbox checked={dontShowAgain} onClick={handleDontShowAgain}>
+              <TextWrapper width={192} height={46}>
+                <Text weight={700} size={14}>
+                  閉じて今後は表示しない
+                </Text>
+              </TextWrapper>
+            </Checkbox>
+          </CheckWrapper>
+        </>
+      ) : (
+        <>
+          <CloseButton>
+            <Icon size={32} icon="close" onClick={handleCloseVideo} />
+          </CloseButton>
+          <VideoWrapper>
+            <Video width=" 1142" height="543" src="https://www.youtube.com/embed/pY2dM-eG5mA" />
+          </VideoWrapper>
+        </>
+      )}
+    </Wrapper>
+  );
+};
+export default WelcomeScreen;
+
+const Wrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  gap: 50px;
+  background: rgba(0, 0, 0, 0.7);
+`;
+
+const InnerWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  padding: 0px;
+  gap: 119px;
+  width: 742px;
+  height: 316px;
+  left: calc(50% - 742px / 2 + 0.5px);
+  top: calc(50% - 316px / 2 - 30.5px);
```

Comment:
I don't think this is doing anything since InnerWrapper isn't being positioned absolutely.

Reply:
I will remove it and test it

Reply:
done

Reply:
a36d7887f9fae43211d25e5df9545ce613bbccec